release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -377,6 +377,34 @@ and no other Reports or ReportQueries depend on the expiring Report, the Meterin
 removes the Report from your cluster at the end of its retention period. For more information,
 see metering Reports xref:../metering/reports/metering-about-reports.adoc#metering-expiration_metering-about-reports[expiration].
 
+[id="ocp-4-6-monitoring"]
+=== Monitoring
+
+[id="ocp-4-6-monitoring-for-user-defined-projects"]
+==== Monitoring for user-defined projects
+
+In {product-title} 4.6, you can enable monitoring for user-defined projects in addition to the default platform monitoring. You can now monitor your own projects in {product-title} without the need for an additional monitoring solution. Using this new feature centralizes monitoring for core platform components and user-defined projects.
```

Comment:
Using this new feature centralizes monitoring for **both** core platform components and user-defined projects.

Add both can make it more clear

Reply:
Thank you for your suggestion. My view is that the word 'both' is superfluous in this sentence.